.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,54 @@
+name: Upload geocat-viz to PyPI
+on:
+  release:
+    types:
+      - published
+jobs:
+  test-build:
+    if: github.repository == 'NCAR/geocat-viz'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install setuptools setuptools-scm wheel twine check-manifest
+      - name: Build tarball and wheels
+        run: |
+          python setup.py sdist bdist_wheel
+          python -m pip wheel . -w dist --no-deps
+      - name: Test the artifacts
+        run: |
+          python -m twine check dist/*
+  publish:
+    needs: test-build
+    if: startsWith(github.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install setuptools setuptools-scm wheel twine check-manifest
+      - name: Build tarball and wheels
+        run: |
+          python setup.py sdist bdist_wheel
+          python -m pip wheel . -w dist --no-deps
+      - name: Test the artifacts
+        run: |
+          python -m twine check dist/*
+      - name: Publish package to PyPI
+        uses: pypa/gh-action-pypi-publish@v1.4.2
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
Do not forget to change "PYPI_PASSWORD" to "PYPI_GEOCAT_VIZ"

Reply:
Ahh yeah, we knew this was going to be a thing. We changed the directory structure, so I think the way we import util functions from viz needs to change. I'm not sure why it's yelling at you from the init in viz, but I think that's what's going on.

The version of geocat-viz is pinned in geocat-examples, though, so this release wouldn't immediately break -examples and we could make the necessary changes before unpinning it.